packages/lesswrong/lib/collections/arbitalTagContentRels/schema.ts

```diff
@@ -0,0 +1,65 @@
+import { schemaDefaultValue } from '../../utils/schemaUtils';
+
+const schema: SchemaType<'ArbitalTagContentRels'> = {
```

Comment:
I think probably all the `canCreate` values, if we have publicly-accessible mutations at all, should be `admins`, not `members`